<!DOCTYPE html>

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-<head>
+<head><!-- -->
```

Comment:
Good job with the document structure for the most part. Make sure the file is named properly. There also needs to be a heading in the head.